test-network/utils.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
```

Comment:
I like this Idea. May rename this file scriptUtils so that folks are aware at first glance that there is no blockchain content in this file

Reply:
Done